### DISCUSSION TITLE
Change background color of IIIF image viewer

### DISCUSSION THREAD

**Discussion by salger1** [thegetty/quire#906] [Get Help] [answered]
We have the image viewer background set to white but when we add IIIF images to the viewer the viewer changes to a black background. 

The first two images on [this page](https://wwwstag.nelson-atkins.org/starr1/Continental/Seventeenth-Century/F58-60-103/) are IIIF and display with a black background but the third is not IIIF and displays on a white background. Is there somewhere we can update the color of the IIIF viewer? 

This is what we have on the variables.scss: 

```
// Image viewer (pop-up figure viewer & entry page viewer)
// -----------------------------------------------------------------------------
$quire-entry-header-height: 4.5rem;

$quire-entry-image-color: $white;
$quire-entry-image-icons-color: $white;
```

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @salger1, we've come across this before. The black background is erroneously caused by a third party dependency (Canvas Panel - a web component that renders a IIIF Canvas) which is somehow overwriting the `variables.scss` file. To fix it, you need to add the line of code below to your `custom.css` which applies the change to the root of the project. Please let us know if it works. 

```css
:root {
  --atlas-background: white;
}
```

**Reply by salger1** [reply]
Looks like that did the trick! Thank you Erin!

**Reply by Erin-Cecele** [reply]
Wonderful news!
